code/game/machinery/stasis.dm

```diff
@@ -52,6 +52,9 @@
 		stasis_enabled = !stasis_enabled
 		stasis_can_toggle = world.time + STASIS_TOGGLE_COOLDOWN
 		playsound(src, 'sound/machines/click.ogg', 60, TRUE)
+		user.visible_message("<span class='notice'>\The [src] [stasis_enabled ? "powers on" : "shuts down"].</span>", \
```

Comment:
fuck, actually second thought, borgs toggling it from the edge of their screen would be a bit silly with this. feel free to fix it yourself, or i'll hit it in a few days.